include/pokemon.h

```diff
@@ -3,6 +3,99 @@
 
 #include "sprite.h"
 
+// Flags for Get(Box)MonData / Set(Box)MonData
```

Comment:
While we're here we should probably call these values or something, given they're not really flags.